api/api.go

```diff
@@ -292,3 +292,44 @@ func (c *Client) Skip(language, slug string) error {
 
 	return errors.New(pe.Error)
 }
+
+// Status sends a request to exercism to fetch the user's
+// completion status for the given language track.
+func (c *Client) Status(language string) ([][]string, error) {
+	url := fmt.Sprintf("%s/tracks/%s/status?key=%s", c.XAPIHost, language, c.APIKey)
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := c.Do(req, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	defer res.Body.Close()
+	if res.StatusCode == http.StatusNoContent {
```

Comment:
So a 204 means that it's all good but you haven't done anything? That seems a little bit cryptic.